wazo_webhookd/plugins/subscription/service.py

```diff
@@ -87,6 +91,7 @@ def create(self, subscription):
         with self.rw_session() as session:
             new_subscription = Subscription(**subscription)
             session.add(new_subscription)
+            session.flush()
```

Comment:
This fix should be easily testable with integration tests, no?

Reply:
done